caffe2/sgd/learning_rate_op.cc

```diff
@@ -163,8 +163,10 @@ C10_EXPORT_CAFFE2_OP_TO_C10_CPU(
     "int? inactive_period = -1, "
     "int? max_iter = -1, "
     "int? num_iter = 0, "
+    "float? decay = 1.0, "
     "float? start_multiplier = 0, "
     "float? end_mulitplier = 0, "
+    "float? end_multiplier = 0, "
```

Comment:
wait, why we double end_multiplier?

Reply:
I had this because I thought that deleting the original typo'ed end_multiplier would break backwards compatibility; is it safe to just rename, or is there something else that has to be done?